manifests/g/GPSoftware/DirectoryOpus/12.31/GPSoftware.DirectoryOpus.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.2.3 $debug=QUSU.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GPSoftware.DirectoryOpus
+PackageVersion: "12.31"
+PackageLocale: en-US
+Publisher: GPSoftware
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: Directory Opus
+PackageUrl: https://www.gpsoft.com.au
+License: Copyright (c) GPSoftware
```

```suggestion
License: Proprietary
```
A copyright isn't a license.